Dockerfile

```diff
@@ -6,6 +6,6 @@ RUN apt-get -y update && apt-get install -y -q build-essential
 ADD jars jars
 ADD consumer.properties.template .
 ADD run_kcl.sh .
-ADD kinesis-to-firehose kinesis-to-firehose
+ADD kinesis-consumer kinesis-consumer
```

Comment:
I'm somewhat agnostic about the binary name.   For now I made all consumers create binaries called `kinesis-consumer` for convenience.  Also it lets the Makefile, Dockerfile, and consumer.properties.template files be exactly the same.